CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+# How to Contribute to Avalanche
+
```

Comment:
I think we could add a blurb here thanking the community member for being interested in contributing to avalanchego. In general, we should have a welcoming tone in this doc and give off the impression that we respect those who contribute to the codebase on their own time.